infra/src/main/resources/db/migration/prod/V77__add_bulk_transfer_id.sql

```diff
@@ -0,0 +1,10 @@
+alter table publication.split
+  add column bulk_transfer_id int null,
+  add constraint bulk_transfer_id_unique unique (bulk_transfer_id);
+
+alter table publication.split_version
+  add column bulk_transfer_id int null;
+
+create unique index bulk_transfer_state_in_progress_unique
+  on publication.split (bulk_transfer_state)
+  where geoviite.publication.split.bulk_transfer_state = 'IN_PROGRESS';
```

Comment:
Kaksi asiaa:
* Otaksuin, että bulk_transfer_id on uniikki integeri.
* Koska ainoastaan yhden splitin pitäisi olla samaan aikaan in_progressissa, tätä varten voidaan myös muodostaa kantarajoitus osittaisindeksillä. Meillä ei käsittääkseni näitä ollut aiemmin käytössä, joten mielipiteitä/muita ehdotuksia tällaisesta osittaisindeksirajoitteesta?

Reply:
En itse ole varma halutaanko tätä rajoittaa ihan kannan tasolla. Geoviitteen puolestahan ei oikeastaan ole väliä montako massasiirtoa on menossa, vaan jos sellainen rajoite on olemassa, niin se on puhtaasti Ratkon rajoite. Esimerkiksi joku Tampereen alueella meneillään olevan massasiirronhan ei pitäisi vaikuttaa mitään esim. johonkin Joensuun alueen kohteisiin, joten siellä teoriassa voisi hyvin olla menossa toinen siirto jos Ratko sen vaan sallisi

Reply:
Joo samaa mieltä Juusen kanssa (eli Ratkon erikoisuus -> riittää kun hanskataan ratko servicen tasolla)

Reply:
Ulkoisen id:n tyypin voinee varmistaa swaggerista. Jos sieltä tulee int niin hyvä näin.